woof-distro/x86_64/slackware64/s15pup64/_00build.conf

```diff
@@ -0,0 +1,304 @@
+#
+#  persistent configuration options
+#
+#  see also DISTRO_SPECS DISTRO_PET_REPOS DISTRO_COMPAT_REPOS-*
+#
+#  **NOTE**: check the original file every once in a while
+#            settings might be added or removed...
+#
+
+# 2createpackages. binaries are usually already stripped. set to 'no' to speed up process
+STRIP_BINARIES=no
+
+## UnionFS: aufs or overlay
+UNIONFS=aufs
+
+#xorg-autoconf from rizalmart - changes perms on /usr/sbin/xorg-autoconf to enable; default is unset
+XAUTOCONF=yes
+
+## Kernel tarballs repo URL for choosing/downloading kernel
+#KERNEL_REPO_URL=http://distro.ibiblio.org/puppylinux/huge_kernels
+
+## Kernel tarball URL
+## avoid being asked questions about downloading/choosing a kernel
+#KERNEL_TARBALL_URL=http://distro.ibiblio.org/puppylinux/huge_kernels/huge-3.14.55-slacko_noPAE.tar.bz2
+
+## an array of generically named programs to send to the ADRIVE, FDRIVE, YDRIVE
+## ADRV_INC="abiword gnumeric goffice"
+ADRV_INC=""
+## YDRV_INC=""
+YDRV_INC=""
+## FDRV_INC="" #this one is very experimental and it's recommended to be left unset
+FDRV_INC=""
+
+## Include kernel-kit generated FDRIVE
+## set to yes or no or leave commented to be asked the question at build time
+#KFDRIVE=no
+
+## build devx? yes/no - any other value = yes
+BUILD_DEVX=yes
+# build docx, nlsx? yes/no - any other value = no
+BUILD_DOCX=no
+BUILD_NLSX=yes
+
+## include devx SFS in ISO?
+DEVX_IN_ISO=no
+
+## packages to build from source
+PETBUILD_GTK=2
+PETBUILDS="busybox aaa_pup_c gtkdialog jwm puppy_flat_icons puppy_standard_icons gtk_theme_flat_grey_rounded gtk_theme_gradient_grey gtk_theme_polished_blue xdg-puppy-jwm firewallstatus powerapplet_tray mtpaint"
```

Comment:
There are so many more that you can use instead of ancient .pet packages ... xlockmore, rxvt-unicode, transmission, Xdialog, yad, netmon_wce ... the list is long. Some petbuilds carry important fixes if you care about CPU usage, security and battery life - for example, the xlockmore petbuild supports power saving while the old .pet packages don't.